src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1519,13 +1542,35 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return terminalPosition;
     }
 
-    // clang-format off
+    // Method Description:
+    // - Returns the number of clicks that occurred (double and triple click support)
+    // Arguments:
+    // - clickPos: the (x,y) position of a given cursor (i.e.: mouse cursor).
+    //    NOTE: origin (0,0) is top-left.
+    // - clickTime: the timestamp that the click occurred
+    // Return Value:
+    // - number of clicks if the new click was at the same location within the time delta
+    const unsigned int TermControl::_NumberOfClicks(winrt::Windows::Foundation::Point clickPos, Timestamp clickTime) const

```

Comment:
Notes from o*line:

```
// Updates global click state etc etc
// Return Value:
// - if the click is in the same position as the last click and within the timeout, the number of clicks within that time window
// - otherwise, 1
const unsigned int TermControl::_incrementClicksWithPosition(Point clickPos, Timestamp clickTime)
```